kolibri/plugins/facility/assets/src/views/FacilityAppBarPage.vue

```diff
@@ -30,6 +31,11 @@
         type: String,
         default: null,
       },
+      appearanceOverrides: {
+        type: Object,
+        required: false,
+        default: null,
+      },
```

Comment:
We should probably do this in all of our `*AppBarPage` components and establish it as a convention for page-specific styling for each plugin.